supplementary_style_guide/glossary_terms_conventions/general_conventions/t.adoc

```diff
@@ -45,6 +45,18 @@
 
 *See also*:
 
+[discrete]
+[[technology-preview]]
+==== image:images/yes.png[yes] Technology Preview (noun)
+*Description*: Technology Preview documentation guidelines apply to any form of early access feature that is not fully supported. Documentation for a Technology Preview feature might be incomplete or include only basic installation and configuration information.
```

Comment:
Same comment as in the other file - Can you remove the first sentence because it's not quite accurate. And instead update with a quick definition of what a TP feature is. 

My suggestion below was this, but whatever you use down there, you should be able to use here too.

```
Technology Preview features provide early access to upcoming product innovations, enabling customers to test functionality and provide feedback during the development process. However, these features are not fully supported. Documentation for a Technology Preview feature might be incomplete or include only basic installation and configuration information.
```

Reply:
Done